MBINCompiler/Models/Structs/GcProjectileData.cs

```diff
@@ -22,28 +22,36 @@ public class GcProjectileData : NMSTemplate
         /* 0x2E0 */ public int MiningDamage;
         /* 0x2E4 */ public int Bounces;
         /* 0x2E8 */ public bool HitOnBounce;
-		[NMS(Size = 7, Ignore = true)]
-		/* 0x2E9 */ public byte[] Padding2E9;
+        [NMS(Size = 7, Ignore = true)]
+        /* 0x2E9 */ public byte[] Padding2E9;
+
         [NMS(Size = 0x10)]
         /* 0x2F0 */ public string PlayerDamage;
         /* 0x300 */ public Colour Colour;
-		
-		/* Not sure about from here to line 36*/ 
-		/* 0x310 */ public int BehaviourFlags;
-		public string[] BehaviourFlagsValues()
```

Comment:
Even if the BehaviourFlagsValues are in the wrong spot this seems like it's necessary?  Weird...